homework2/task1.py

```diff
@@ -0,0 +1,7 @@
+userInput = input("Please write boolean value(true/false): ")
```

Comment:
პითონში სტანდარტია snake_case 
```suggestion
user_input = input("Please write boolean value(true/false): ")
```